openghg/retrieve/_access.py

```diff
@@ -67,14 +67,19 @@ def get_obs_surface(
     if not obs_results:
         raise ValueError(f"Unable to find results for {species} at {site}")
 
-    retrieved_data: Union[ObsData, None] = obs_results.retrieve(site=site, species=species, inlet=inlet)  # type: ignore
-    
+    retrieved_data: Union[ObsData, List[ObsData], None] = obs_results.retrieve(site=site, species=species, inlet=inlet)
+
     if retrieved_data is None:
         return None
+    elif isinstance(retrieved_data, list):
+        print("No data returned.")
```

Comment:
Is this supposed to be above?

Reply:
No - this was implementing the logic where we only expect `get_obs_surface()` to return **one Dataset**. This is the first line of a set of lines that are printed out to explain why no data was returned and what extra details to include to get this to return something.

Reply:
Ah OK yes, makes sense now.